From 69a64b4426fdeff1c1e098941496deda5ad3ea31 Mon Sep 17 00:00:00 2001
From: Scott Murray <scott.murray@konsulko.com>
Date: Thu, 6 Oct 2022 11:56:38 -0400
Subject: [PATCH 1/5] Make Boost requirements more liberal

To allow building with Yocto Project 4.0/kirkstone, remove the EXACT
specifier from the Boost find_package call and restore the lower
Boost version originally specified before upstream commit 7ea6026d
raised it due to issues with the in-tree build.

Upstream-Status: Pending

Signed-off-by: Scott Murray <scott.murray@konsulko.com>
---
 kuksa-val-server/boost.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kuksa-val-server/boost.cmake b/kuksa-val-server/boost.cmake
index 7358976..0da54df 100644
--- a/kuksa-val-server/boost.cmake
+++ b/kuksa-val-server/boost.cmake
@@ -19,14 +19,14 @@
 # 4. Change version in main README.md file for KUKSA.val Server
 
 set(Boost_USE_STATIC_LIBS OFF)
-set(BOOST_VER 1.82.0)
+set(BOOST_VER 1.75.0)
 set(Boost_NO_BOOST_CMAKE ON)
 set(BOOST_COMPONENTS filesystem program_options system log thread)
 ADD_DEFINITIONS(-DBOOST_LOG_DYN_LINK)
 
 # Workaround function to allow cmake call `find_package` twice. Avoide side effects from local variables, which are produced be `find_package`
 function(findBoost Required)
-    find_package(Boost ${BOOST_VER} EXACT ${Required} 
+    find_package(Boost ${BOOST_VER} ${Required}
         COMPONENTS ${BOOST_COMPONENTS}
         OPTIONAL_COMPONENTS unit_test_framework
     )
-- 
2.41.0

